.devcontainer/devcontainer.json

```
@@ -28,9 +31,10 @@
                 "johnstoncode.svn-scm",
                 "ms-vscode.cpptools",
                 "MS-vsliveshare.vsliveshare",
-                "natqe.reload"
+                "natqe.reload",
+                "vadimcn.vscode-lldb"
             ]
         }
     },
-    "postCreateCommand": "find . -wholename '*.git*'  -type d -prune -o -type f -exec chown vscode:vscode {} \\; && sh /workspaces/r-dev-env/scripts/localscript.sh"
+    "postCreateCommand": "find . -wholename '.git*' -type d -prune -o -type f -exec chown vscode:vscode {} \\; && \\\n  gcc -shared -fPIC -o /workspaces/r-dev-env/scripts/allow_ptrace.so /workspaces/r-dev-env/scripts/allow_ptrace.c && \\\n  chmod +x /workspaces/r-dev-env/scripts/launch_r.sh && \\\n  sh /workspaces/r-dev-env/scripts/localscript.sh"
```

It is a good idea to compile `allow_ptrace.c` and change permissions on `launch_r.sh` as part of the setup. But we should avoid hard-coding the paths as much as possible as the root is different on GitPod and will be different if someone clones the repo to a folder named something other than `r-dev-env`. I would try to do these steps as part of `localscript.sh` and then use a relative path:
```suggestion
   "postCreateCommand": "find . -wholename '.git*' -type d -prune -o -type f -exec chown vscode:vscode {} \\; && \\\n  sh ./scripts/localscript.sh"
```

If this doesn't work, maybe we can use `$PWD/scripts/localscript.sh` instead.